src/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -9,27 +9,34 @@ import dataArray from "../../data";
 
 const SearchResultsPage = () => {
   const [films, setFilms] = useState(dataArray);
-  const [isFilmActive, setIsFilmActive] = useState(false);
-  const [activeFilm, setActiveFilm] = useState();
-  const len = films[films.length - 1].id;
+  const [activeFilm, setActiveFilm] = useState(null);
+
+
+  const handleSetFilm = film => {

```

Comment:
it's better to rename it to handleSetFilms as total FILMS is updated, sorry for this mistake